runtime/queries/nix/highlights.scm

```diff
@@ -1,81 +1,84 @@
 (comment) @comment
 
 [
-  "if"
+  "if" 
   "then"
   "else"
   "let"
   "inherit"
   "in"
   "rec"
-  "with"
+  "with" 
   "assert"
+  "or"
 ] @keyword
 
 ((identifier) @variable.builtin
- (#match? @variable.builtin "^(__currentSystem|__currentTime|__nixPath|__nixVersion|__storeDir|builtins)$")
+ (#match? @variable.builtin "^(__currentSystem|__currentTime|__nixPath|__nixVersion|__storeDir|builtins|false|null|true)$")
  (#is-not? local))
```

Comment:
could we revert the change here and on L36-L39? I think the `@constant.builtin(.boolean)` captures work better on true/false/null